server/game/cards/attachments/07/catapultonthewall.js

```diff
@@ -0,0 +1,40 @@
+const DrawCard = require('../../../drawcard.js');
+
+class CatapultOnTheWall extends DrawCard {
+    setupCardAbilities(ability) {
+        this.action({
+            title: 'Kneel Catapult and attached character',
+            condition: () => this.game.currentChallenge,
+
+            // This is not a nice interaction for users, but `ability.costs` would need to be updated with a different method to allow something like attachment.parent.kneel() (I think)
+            cost: [
+                ability.costs.kneelSelf(),
+                ability.costs.kneel(card => card === this.parent)
```

Comment:
If you pull in this commit, you should be able to get rid of the weird interaction:
https://github.com/ystros/throneteki/commit/f998dd75cc01a80e40338dc692c24a5296cbd4a4